apps/prairielearn/src/ee/pages/instructorAiGenerateQuestion/instructorAiGenerateQuestion.ts

```diff
@@ -120,7 +120,13 @@ router.post(
           ),
         );
       } else {
-        throw new error.HttpStatusError(500, `Job sequence ${result.jobSequenceId} failed.`);
+        if (req.get('HX-Request')) {
+          res
+            .set('HX-Redirect', res.locals.urlPrefix + '/jobSequence/' + result.jobSequenceId)
+            .send();
+        } else {
+          res.redirect(res.locals.urlPrefix + '/jobSequence/' + result.jobSequenceId);
+        }
```

Comment:
Rather than 500ing, I figured it was better to redirect to the server job logs. Do you still want to expose these to users?

IMO we should update `GenerationResults` to render a link to the logs page on failure; a page-wide redirect is a bit jarring. @jlevine18 can you take that on in another PR?

Reply:
will do

Reply:
See #10424.